cypress/support/step_definitions/dictionary/addConceptInDictionaryPage/addconceptInDictioanryPage.ts

```diff
@@ -0,0 +1,74 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+  Given,
+  Then,
+  When
+} from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from "../../../utils";
+
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+Then('the user selects "Import existing concept"', () => {
```

Comment:
This on should be `When` not `Then`
```suggestion
When('the user selects "Import existing concept"', () => {
```